__tests__/app.test.js

```diff
@@ -98,6 +107,54 @@ describe("/api/articles/:article_id",()=>{
     })
 })
 
+describe("/api/articles/:article_id/comments",()=>{
+    test("GET 200 and the requested article by id", () => {
+        return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+                .then(({body})=>{
+                    const {comments} = body
+                    comments.forEach(comment => {
```

Comment:
don't forget a length check if you're iterating to prevent a false positive on this test